core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/HealthMonitorSpec.scala

```diff
@@ -13,7 +13,7 @@ import org.broadinstitute.dsde.rawls.monitor.HealthMonitor._
 import org.mockito.ArgumentMatchers._
 import org.mockito.Mockito._
 import org.scalatest.concurrent.{Eventually, ScalaFutures}
-import org.scalatest.mock.MockitoSugar
+import org.scalatest.mockito.MockitoSugar
```

Comment:
Might as well leave it if it's not using anything in `MockitoTestUtils` (but will fix `mock` -> `mockito`).